src/main/java/com/develhope/java10/hellotests/HelloTestsApplication.java

```diff
@@ -7,29 +7,20 @@
 
 public class HelloTestsApplication {
 	public static void main(String[] args) {
-		File file = new File("C:\\Develhope\\HelloTests\\src\\main\\java\\com\\develhope\\java10\\hellotests\\test.txt");
+		// federico ha problemi con il pathname! Funziona solo con absoultepath
+		File file = new File("C:\\Users\\Federico\\Desktop\\HelloTests\\src\\main\\java\\com\\develhope\\java10\\hellotests\\test.txt");
 		List<String> stringList = new ArrayList<>();
-		String stringa = "";
 		try{
 			BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
 			String strCurrentLine;
 			while ((strCurrentLine = bufferedReader.readLine()) != null) {
 				stringList.add(strCurrentLine);
 			}
-
-		} catch (IOException e) {
-			throw new RuntimeException(e);
-		}
-		try {
-
-			BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
-			while(bufferedReader.readLine()!=null){
-
-			}
 			bufferedReader.close();
 		} catch (IOException e) {
 			throw new RuntimeException(e);
 		}
+
```

Comment:
toogliere il +

Reply:
quello non è un "+" è uno spazio vuoto che prima non ci stava e git ti dice che ci stà in + una linea vuota, è semplicemente un invio in più che mi sono ritorvato non è presente nel codice